src/ZoneServer/Commands/ChatCommands.Handlers.cs

```diff
@@ -754,6 +755,51 @@ private CommandResult HandleLevelUp(Character sender, Character target, string m
 			return CommandResult.Okay;
 		}
 
+		/// <summary>
+		/// Levels up target's job level.
+		/// </summary>
+		/// <param name="sender"></param>
+		/// <param name="target"></param>
+		/// <param name="message"></param>
+		/// <param name="command"></param>
+		/// <param name="args"></param>
+		/// <returns></returns>
+		private CommandResult HandleJobLevelUp(Character sender, Character target, string message, string command, Arguments args)
+		{
+			var levels = 1;
+			if (args.Count >= 1 && (!int.TryParse(args.Get(0), out levels) || levels < 1))
+				return CommandResult.InvalidArgument;
+
+			var jobLevelsGained = target.JobLevelUp(levels);
+
+			if (jobLevelsGained == 0)
+			{
+				if (sender == target)
+				{
+					sender.ServerMessage(Localization.Get("Your job level can not be changed."));
```

Comment:
I left that, because if you are, let's say job level 30 and you use the command /joblevelup 20, the command will fail, because you exceeding the cap, so technically the job level could not be changed, even thought it could be raised further. I guess we could default to the highest available level, but I didn't want to overcomplicate the command with additional calculations?

Reply:
Oh, you're right. I missed that the EXP db will throw in that case. But we'll never reach this message then. I would also say that we probably don't want commands to throw, so I'm going to add a fix for this.